blueprints/banking_assistant/__init__.py

```diff
@@ -0,0 +1,498 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.core import FormEntity
+import random
+import json
+import os
+
+app = Application(__name__)
+
+__all__ = ["app"]
+
+# User data class and helper functions for handlers
+
+user = None
+
+
+class User:
+    def __init__(self, request):
+        self.sample_users = {
+            "johndoe123": 0,
+            "larry_l12": 1,
+            "splashbro30": 2,
+        }
+        user = self._pull_data(request)
+        directory = os.path.dirname(os.path.abspath(__file__))
+        path = os.path.join(directory, "data/sample_user_data.json")
+        with open(path) as f:
+            data = json.load(f)
+            self.data = data[user]
+
+    def _pull_data(self, request):
+        if request.context.get("user_name"):
+            user_name = request.context.get("user_name")
+            self.user = self.sample_users[user_name]
```

Comment:
It might be possible that the username in the context is not in the list of sample users. Should we gracefully handle this case?